build.gradle

```
@@ -84,6 +84,15 @@ subprojects {
     tasks.withType(Test) {
         systemProperty 'recreate', System.getProperty('recreate', 'false')
     }
+
+    checkUnusedDependencies {
+        // avoid spurious flags
+        ignore 'org.jetbrains', 'annotations'
+    }
+
+    tasks.check.dependsOn(checkImplicitDependencies)
+    tasks.check.dependsOn(checkUnusedDependencies)
+    tasks.check.dependsOn(checkUnusedConstraints)
```

`checkUnusedConstraints` is a dependency of `check` by default

https://github.com/palantir/gradle-consistent-versions/blob/980bca5499e558e0e96b02df36a527053e80bf31/src/main/java/com/palantir/gradle/versions/VersionsPropsPlugin.java#L93

```suggestion
    tasks.check.dependsOn(checkImplicitDependencies)
    tasks.check.dependsOn(checkUnusedDependencies)
```